src/index.html

```diff
@@ -11,7 +11,212 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    class="form">
+      <fieldset class="user_info">
```

Comment:
Use more descriptive class names. for example:
```suggestion
      <fieldset class="form-fieldset">
```